bin/improver-wind-direction

```diff
@@ -53,8 +64,9 @@ def main():
 
     # Returns 3 cubes - r_vals and confidence_measure cubes currently
     # only contain experimental data to be used for further research.
+    bmethod = args.backup_method
     cube_mean_wdir, cube_r_vals, cube_confidence_measure = (
-        WindDirection().process(wind_direction))
+        WindDirection(backup_method=bmethod).process(wind_direction))
```

Comment:
Could we have a new CLI test please for the "first_realization" option?

Reply:
Done